ArtistList/app/src/main/java/com/grzesica/przemek/artistlist/Container/DataFetcherDIBuilder.java

```diff
@@ -0,0 +1,53 @@
+package com.grzesica.przemek.artistlist.Container;
+
+import android.content.Context;
+import android.os.Handler;
+
+import com.grzesica.przemek.artistlist.Model.DataFetcher;
+
+import java.util.concurrent.AbstractExecutorService;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by przemek on 05.03.18.
+ */
+
+public class DataFetcherDIBuilder implements IDataFetcherDIBuilder {
```

Comment:
POdobnie tutaj, to co ustawiasz w metodach powinno byc ustawione jao wartosci domyslne a builder powinien miec mozliwosc ustawiania/modyfikacji pol. 
Tutaj masz przykład: https://dzone.com/articles/design-patterns-the-builder-pattern